schema.graphql

```diff
@@ -437,10 +451,35 @@ Docs group: Match
 """
 type MatchQuestion {
   """
-  Which question number this was. Starts at 1.
+  Which tossup-bonus cycle number this was. Starts at 1.
   """
   question_number: Int!
 
+  """
+  Information about the tossup question in this cycle.
+  """
+  tossup_question: Question
+
+  """
+  The time at which the moderator began reading the tossup. An ISO 8601-formatted string, with time zone, in the tournament's local time zone.
+  """
+  tossup_start_time: Date
```

Comment:
I anticipated that `Tournament.start_date` would often be just a date without time. I guess that sort of contradicts the specification, but I think it will at least sometimes be necessary to omit the time (e.g., when announcing a tournament relatively far in advance). Both the field name and description call it a date (not a time or datetime).

On the other hand, this field here would be useless without a time.

So I think it's fine to keep them named "inconsistently" (because their intended format may not be consistent). Do you buy that?

Reply:
Sure.